packages/node/src/schematics/application/application.spec.ts

```diff
@@ -244,6 +244,39 @@ describe('app', () => {
       );
     });
 
+    it.each`
+      firstAppName     | secondAppName
+      ${'my-frontend'} | ${'my-frontend'}
+      ${'billing'}     | ${'cart'}
+    `(
+      'should configure proxies for multiple frontend projects $firstAppName and $secondAppName',
+      async ({ firstAppName, secondAppName }) => {
+        appTree = createApp(appTree, 'my-frontend');
+
+        appTree = await runSchematic(
+          'app',
+          { name: 'cart', frontendProject: 'my-frontend' },
+          appTree
+        );
+
+        const tree = await runSchematic(
+          'app',
+          { name: 'billing', frontendProject: 'my-frontend' },
+          appTree
+        );
+
+        expect(tree.exists('apps/my-frontend/proxy.conf.json')).toBeTruthy();
+
+        expect(
+          JSON.parse(tree.readContent('apps/my-frontend/proxy.conf.json'))
```

Comment:
nit, but you can use `readJsonInTree` 😃

Reply:
Thank you. Totally forgot about readJsonInTree.